libraries/AP_Common/missing/termios.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include_next <termios.h>
+
+#if defined(__ANDROID__) && defined(__BORLANDC__)
+
+static __inline__ int cfsetspeed(struct termios *s, speed_t  speed)
```

Comment:
I tried, but nothing there, cfsetispeed() and cfsetospeed() are defined directly on API Level 19.

https://github.com/UrusTeam/android_ndk_toolchain_cross/blob/level_19/sysroot/usr/include/termios.h#L85